### PR TITLE
MAQL DML support

### PR DESCRIPTION
**Pull request by sadam21**

### CODE REVIEW

File: src/test/java/com/gooddata/dataset/TaskStateTest.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import java.io.IOException;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class TaskStateTest {
+
+    @Test
+    public void testDeserialize() throws IOException {
+        final String json = "{\n" +
```
Comment (martiner) [inline src/test/java/com/gooddata/dataset/TaskStateTest.java]: please no embedded json

Comment (sadam21) [inline src/test/java/com/gooddata/dataset/TaskStateTest.java]: done

**Comment by martiner** [comment]
looks terrific. one more think: can you please write a simple acceptance test in `ShowcaseAT`?

**Comment by martiner** [comment]
- [ ] update readme

**Comment by sadam21** [comment]
Readme updated, AT added.

**Pull request merged**